backends-clickhouse/src/main/scala/io/glutenproject/utils/CHExpressionUtil.scala

```diff
@@ -95,6 +95,7 @@ object CHExpressionUtil {
   )
 
   final val CH_BLACKLIST_SCALAR_FUNCTION: Map[String, FunctionValidator] = Map(
+    ARRAY_JOIN -> DefaultValidator(),
```

Comment:
@exmy, @zzcclp, please note this change for CH.

Reply:
It's ok.